.github/workflows/unit-tests.yml

```diff
@@ -50,7 +50,11 @@ jobs:
         if: always() && steps.build-babelfishpg_unit.outcome == 'success'
         timeout-minutes: 60
         run: |
-          sudo ~/psql/bin/psql -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 
+          sudo ~/psql/bin/psql -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 2>&1
+          if grep -w "ERROR:  schema \"babelfishpg_unit\" does not exist" ~/psql/output.out; then
+            echo "babelfishpg_unit extension is not installed. Check the stderr for more details."
+            exit 1
+          fi
```

Comment:
Failure condition is quite specific. We should rather use a generic way, I would recommend using `ON_ERROR_STOP` psql parameter instead, which will make sure to return non-zero exit code in case of an error.  
You can specify it to psql command like: `psql -v ON_ERROR_STOP=1 ...`

Additionally, we can do this everywhere we are using psql command in github actions and provide a generic solution.

Reply:
Makes sense. updated in latest rev.